.bazelrc

```diff
@@ -21,6 +21,9 @@ build:system-clang-17 --action_env=CXX=clang++-17 --host_action_env=CXX=clang++-
 build:system-clang-18 --config=system-clang
 build:system-clang-18 --action_env=CC=clang-18 --host_action_env=CC=clang-18
 build:system-clang-18 --action_env=CXX=clang++-18 --host_action_env=CXX=clang++-18
+build:system-clang-19 --config=system-clang
```

Comment:
Curious about the context here -- is this required for parquet or is this just something you found when developing?

Reply:
I upgraded to clang-19 locally, so this is just a way of sneaking support for system clang-19 into the build (I use this for compile_commands.json support).